src/common.js

```diff
@@ -124,12 +124,18 @@ export default class Helpers {
         const targetPath = path.replace(/[\\\/][^\\\/]+$/, '');
         const filename = path.replace(/^.*[\\\/]/, '').replace(/\?(.*)/, '');
         
-        if(!CONFIG.AIE.TEMPORARY.import[path]) {
-          await Helpers.verifyPath("data", `worlds/${game.world.name}/adventures/${adventurePath}/${targetPath}`);
-          const img = await zip.file(path).async("uint8array");
-          const i = new File([img], filename);
-          await Helpers.UploadFile("data", `worlds/${game.world.name}/adventures/${adventurePath}/${targetPath}`, i, { bucket: null })
-          CONFIG.AIE.TEMPORARY.import[path] = true;
+        if(!CONFIG.AIE.TEMPORARY.import[path]) {         
+          await Helpers.verifyPath("data", `worlds/${game.world.data.name}/adventures/${adventurePath}/${targetPath}`);
+          const fileObj = zip.file(path);
+          if (fileObj !== null) {
+            const img = await zip.file(path).async("uint8array");
+            const i = new File([img], filename);
+            await Helpers.UploadFile("data", `worlds/${game.world.data.name}/adventures/${adventurePath}/${targetPath}`, i, { bucket: null })

```

Comment:
Was also thinking about aligning to use game.world.id, which a couple other parts of code use. Feels like that should be consistent and id probably makes the most sense to use.